### ISSUE TITLE
[Elasticsuite] Issue with base64_encoded image used as return of a controller

### ISSUE THREAD

**Issue by romainruaud** [yireo/Yireo_LinkPreload#37]
Hi,

it has been reported to Elasticsuite that this extension _might_ causes some issues on the tracking pixel url of Elasticsuite.

Since this image is generated as base64 encoded result of a layoutless controller, this might be an edge-case your module is not covering. 

Or maybe we could just add some sort of one-liner by our side to allow proper compliance between our two modules.

Let us know

You can check this comment which is well detailed : https://github.com/Smile-SA/elasticsuite/issues/2393#issuecomment-1056734236

Regards

**Comment by jissereitsma** [comment]
Thanks for posting the issue. I find the original thread quite hard to follow, with conclusions made early, traces being typed down instead of actual dumps, and a problem description that does not actually seem to hit the issue. However, without trying to duplicate this myself, I can "guess" that the problem might be the LinkPreload module reacting upon the response, assuming it is an HTML-like response, while it is actually not. I've added a new release 1.4.16 to improve this. If that version could be checked up upon, that would be great.
